src/components/Player/Pages/Overview/Summary.jsx

```diff
@@ -34,19 +35,28 @@ const SummOfRecMatches = ({ matchesData }) => {
 
   const dataKeys = Object.keys(data);
 
-  const numRows = Math.min(MAX_MATCHES_ROWS, matchesData.length);
+  const desiredNumRows = Math.min(MAX_MATCHES_ROWS, matchesData.length);
 
-  for (let i = 0; i < numRows; i += 1) {
+  // XXX - getPlayerRecentMatches seems to only return a maximum of 20 matches
```

Comment:
getPlayerRecentMatches is more performant and therefore we use it here.

getPlayerMatches can access all columns, but you need to declare the ones you want with the `project` query parameter, otherwise it just returns a small default set.